### ISSUE TITLE
Text jumping with line wrap enabled

### ISSUE THREAD

**Issue by asvc** [notepad-plus-plus/notepad-plus-plus#8182]
### Description of the Issue
Text wrapping is glitching on input. Quite annoyingly so.

### Steps to Reproduce the Issue
1. Turn on text wrap.
2. Open some file with long lines and active highlighting
3. Start typing where you don't expect new text to affect the wrapped line

### Expected Behavior
1. Text does not jump on key press

### Actual Behavior
1. Text jumps momentarily on keypress. Please see the video:  

![xx](https://user-images.githubusercontent.com/300146/80490293-4d20f680-89a0-11ea-9bf1-88bdab14554d.gif)

### Debug Information
```
Notepad++ v7.8.6   (64-bit)
Build time : Apr 21 2020 - 15:23:03
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 1909
OS Build : 18363.778
Plugins : BetterMultiSelection.dll mimeTools.dll NppConverter.dll NppExport.dll 
```

**Comment by sasumner** [comment]
From the text that is "jumping" in the sample, I wonder if the *Clickable Link* stuff is interfering?
Nice sample, BTW.

**Comment by asvc** [comment]
@sasumner , you are correct. Is there anything else I can do to help with this?

**Comment by sasumner** [comment]
@asvc 

Can you clarify your statement "you are correct" ?

Does that mean that you have *Clickable Links* enabled?
Most people have the underline feature turned on but I didn't see that in your text so I can't be sure.

If you had *Clickable Links* enabled, if you disable it and repeat the same trial, does the text still jump like before?

**Comment by asvc** [comment]
> Does that mean that you have Clickable Links enabled?
> Most people have the underline feature turned on but I didn't see that in your text so I can't be sure.

Yes, I had it enabled. It is visible in the video (well, gif) albeit only for a short time. Watch closely and you will see it.

> If you had Clickable Links enabled, if you disable it and repeat the same trial, does the text still jump like before?

No, after disabling clickable links text does not jump.
